kiwi/system/uri.py

```diff
@@ -202,24 +221,28 @@ def _local_path(self, path):
         return os.path.normpath(path)
 
     def _obs_project(self, name):
-        return ''.join(['http://download.opensuse.org/repositories/', name])
-
-    def _ibs_project(self, name):
-        ibs_project = 'http://download.suse.de/ibs/'
-        return ibs_project + name.replace(':', ':/')
+        return ''.join(
+            [
+                self.runtime_config.get_obs_download_server_url(),
+                '/repositories/', name
+            ]
```

Comment:
Not sure we have to include repositories here, related to my comment above in defaults.py